src/java.base/share/classes/sun/security/pkcs/PKCS7.java

```diff
@@ -1000,7 +1000,7 @@ public static URI getTimestampingURI(X509Certificate tsaCertificate) {
      * @return the encoded timestamp token
      * @throws IOException The exception is thrown if an error occurs while
      *                     communicating with the TSA, or a non-null
-     *                     TSAPolicyID is specified in the request but it
+     *                     TSAPolicyID is specified in the request, but it
```

Comment:
The previous text w/o a comma reads better to me as it is a complete statement: "or a non-null TSAPolicyID is specified in the request but it does not match the one in the reply".

Reply:
Removing the comma.